templates/articles/software_for_project_quality_management.json

```
@@ -0,0 +1,7 @@
+{
+  "title": "software_for_project_quality_management",
+  "author": "Melanie Key",
+  "date": "2023-10-15",
+  "subpath": "software_for_project_quality_management",

```

The subpath is the name of your HTML file, so this should be:
```suggestion
  "subpath": "software_for_project_quality_management.html",
```